src/components/LineContent/index.tsx

```diff
@@ -53,11 +54,10 @@ export default class LineContent extends Component<any, any> {
         }
 
         return (
-            <span className={lineContent} style={style}>
-                {data &&
-                    data.map((part, n) => (
-                        <LinePart part={part} format={formatPart} key={`line-${number}-${n}`} />
-                    ))}
+            <span className={lineContent} style={style} onMouseUp={onMouseUp}>
+                {data && data.map((part, n) => (
+                    <LinePart part={part} format={formatPart} key={`line-${number}-${n}`} style={style} />
+                ))}
```

Comment:
The diff is still weird here. The original has `key` but not `style` for `<LinePart />`. The updates has `style` and not `key`. Also, have you tested that the changes you are making here doesn't break your SOC update?

Reply:
It seems like key is still present here?

Reply:
Ah you're right. But `style` is on the `<span />`, so I don't think it should be applied to the `<LinePart />`?

Reply:
I had each of these inline styled because I wanted them each to take up the whole width of the logs container (so we can click anywhere on the line to close the metadata). They each need the same style (inline, 100% width). So we could pass it through 🤷🏾‍♂️ or not.

Reply:
The span should inherit the style of the child, but this is probably fine.